source/core/index.ts

```diff
@@ -1330,6 +1333,17 @@ export default class Request extends Duplex implements RequestEvents<Request> {
 		delete options.request;
 		delete options.timeout;
 
+		const requestOptions = options as any; // Temporary workaround
```

Comment:
```suggestion
		// Temporary workaround
		const requestOptions = options as any;
```

Reply:
That's really a temp thing, before the merge (if any) it well be replaced with something better.